src/pages/home/Home.jsx

```diff
@@ -1,7 +1,180 @@
-import React from "react";
+import "./Home.css";
+import smiley from "../../assets/smiley.svg";
+import illustration_scroll from "../../assets/illustration_scroll.svg";
+import illustration_network from "../../assets/illustration_network.svg";
```

Comment:
Suggestion: you can import both illustration in one file and use it from there

Reply:
done